sematic/cli/logs.py

```diff
@@ -0,0 +1,98 @@
+# Standard Library
+import logging
+import time
+
+# Third-party
+import click
+
+# Sematic
+from sematic import api_client
+from sematic.cli.cli import advanced, cli
+from sematic.config.config import get_config, switch_env
+from sematic.log_reader import (
+    ObjectSource,
+    line_stream_from_log_directory,
+    load_log_lines,
+)
+
+DEFAULT_LOG_LOAD_MAX_SIZE = 1000
+
+
+@cli.command("logs", short_help="Read logs for a run")
+@click.argument("run_id", type=click.STRING)
+def logs(run_id: str):
+    """Read the logs for the run directly from blob storage, and print to stdout.
+
+    If the run is still producing logs, this will follow the logs "live" until
+    no more log lines are being produced.
```

Comment:
Should we instead by default only display currently available logs and return the prompt, and have a `-f` option to follow logs? Pretty standard.

Reply:
I second this request.

Reply:
done